docs/design/initial-design.md

```diff
@@ -119,6 +120,44 @@ status:
     accessModes:
     - ReadWriteMany
     storage: 10Gi
+  phase: Completed
+
+```
+
+### `DataMoverVolumeBackup`
+
+The `DataMoverVolumeBackup` represents the backup operation for a
+single volume. This resource will be created by the Data Mover
+Controller, one for each volume being backed up in the
+`DataMoverBackup` operation. StorageClassName, AccessModes, and
+Storage are pulled from the original PVC (if not available, can this
+be inferred from the CSI snapshot? If not, figure out default -- in
```

Comment:
Looking at the API, I'm not sure we'd be able to get all this info from the CSI snapshot.

Reply:
That was my concern as well. Is it OK to require that the PVC still exists? Currently the proposal assumes choosing PVCs not CSI snapshots. One problem with choosing snapshots is the user might have to produce the missing metadata (storageclass, size, etc.). Another problem is we'd want to avoid the namespace/label approach because there could be multiple versions of the same underlying volume represented there. Maybe for now we just limit it to existing PVCs?